### DISCUSSION TITLE
YimMenu, GTA V.  Rockstar could process this transaction.

### DISCUSSION THREAD

**Discussion by HGL14** [YimMenu/YimMenu#3614] [Q&A]
Every time I inject YimMenu this error code comes up, nothing happens when i press enter. I'm stuck in the error message, how do I fix this?

https://gyazo.com/27e289f64e07354dd1c2ad92e6b7587d

**Comment by lonelybud** [comment]
may be related https://github.com/YimMenu/YimMenu/issues/3615. delete your settings `%appdata%/YimMenu/settings.json`

**Reply by HGL14** [reply]
Thanks, turning off "disabled phone" solved the issue.
